app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -98,125 +75,35 @@ function ReviewIssuesStep({
-      return (
-        <>
-          <Warning
-            heading={
-              isFailedDocType
-                ? t('errors.doc_auth.doc_type_not_supported_heading')
-                : t('errors.doc_auth.rate_limited_heading')
-            }
-            actionText={t('idv.failure.button.warning')}
-            actionOnClick={onWarningPageDismissed}
-            location="doc_auth_review_issues"
-            remainingAttempts={remainingAttempts}
-            troubleshootingOptions={
-              <DocumentCaptureTroubleshootingOptions
-                location="post_submission_warning"
-                showAlternativeProofingOptions={!isFailedResult}
-                showSPOption={false}
-                heading={t('components.troubleshooting_options.ipp_heading')}
-              />
-            }
-          >
-            <UnknownError
-              unknownFieldErrors={unknownFieldErrors}
-              remainingAttempts={remainingAttempts}
-              isFailedDocType={isFailedDocType}
-            />
-
-            {!isFailedDocType && remainingAttempts <= DISPLAY_ATTEMPTS && (
-              <p>
-                {formatWithStrongNoWrap(
-                  t('idv.failure.attempts_html', { count: remainingAttempts }),
-                )}
-              </p>
-            )}
-          </Warning>
-          <Cancel />
-        </>
-      );
-    }
-    // ipp enabled
+    // Warning(try again screen)
     return (
-      <Warning
-        heading={
-          isFailedDocType
-            ? t('errors.doc_auth.doc_type_not_supported_heading')
-            : t('errors.doc_auth.rate_limited_heading')
-        }
-        actionText={t('idv.failure.button.try_online')}
-        actionOnClick={onWarningPageDismissed}
-        location="doc_auth_review_issues"
+      <DocumentCaptureWarning
+        isFailedDocType={isFailedDocType}
+        isFailedResult={isFailedResult}
         remainingAttempts={remainingAttempts}
-        troubleshootingOptions={
-          <DocumentCaptureTroubleshootingOptions
-            location="post_submission_warning"
-            showAlternativeProofingOptions={!isFailedResult}
-            heading={t('components.troubleshooting_options.ipp_heading')}
-          />
-        }
-      >
-        {!isFailedDocType && <h2>{t('errors.doc_auth.rate_limited_subheading')}</h2>}
-        <UnknownError
-          unknownFieldErrors={unknownFieldErrors}
-          remainingAttempts={remainingAttempts}
-          isFailedDocType={isFailedDocType}
-        />
-
-        {!isFailedDocType && remainingAttempts <= DISPLAY_ATTEMPTS && (
-          <p>
-            {formatWithStrongNoWrap(t('idv.failure.attempts_html', { count: remainingAttempts }))}
-          </p>
-        )}
-      </Warning>
+        unknownFieldErrors={unknownFieldErrors}
+        actionOnClick={onWarningPageDismissed}
+      />
     );
   }
-  // hasDismissed = true
+  // Show review issue screen, hasDismissed = true
```

Comment:
It should be safe to make this part of the code like `if (hasDisimssed) {...}` but I'm not sure it's a good idea since it removes our catch-all render.

Reply:
Generally this warning screen etc is not consistent in displaying error messages maybe for it's long history. I am thinking create a design ticket to review the error messages in ipp/noipp scenarios.
In general we have a few flags/conditions

1. IPP or non IPP
2. doc_auth_result failed or not failed
3. doc_type supported or not supported.


The control logic and screen text is added piece by piece in a long history. For example, the warning page button shows "Try again" or "Try again online", which really makes no difference to me.